task/anchore-cli/0.1/anchore-cli.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
can we have tests for this tast?

Reply:
not sure if we can have tests for this task